Fix #2530 Setting constant colors for glyphs is non-intuitive

StasJ opened the pull request:
Fix #2530

Adds an explicit "Use Constant Color" checkbox and color selector to all the mentioned renderers.

clyne reviewed (commented):
In the case of the barb rendering, turning on constant color causes the TFE to be hidden. However, for the wireframe, contour, and flow renderer the TFE is not hidden. Is there a reason to not be consistent?

StasJ commented:
> In the case of the barb rendering, turning on constant color causes the TFE to be hidden. However, for the wireframe, contour, and flow renderer the TFE is not hidden. Is there a reason to not be consistent?

That was something I added when refactoring the GUI because in the barb renderer, when constant color is enabled, the color mapping var is not used at all which is not the case in the other renderers. I changed it to make it consistent, however.

clyne commented:
@stasj needs to resubmit to resolve issue with 26 files change from transition from master to main.
